model/dimention_option.go

```diff
@@ -0,0 +1,20 @@
+package model
+
+// LinkObject represents a generic structure for all links
+type LinkObject struct {
```

Comment:
I would call it Link rather as Golang does not have `objects` as found on C#/C++/Java/Scala.

Reply:
done